gettext/lua_xgettext.py

```diff
@@ -29,9 +31,9 @@ class Lua_GetText(object):
     _SIMPLE_STRING = re.compile(
         r'''_\(        #parenthese must follow _
```

Comment:
The fact that this breaks on `gettext()` explains some untranslated strings. I forget what a non-matching string is in the Python re module, probably something like `(?:_|gettext)`